src/gui/guiFormSpecMenu.cpp

```diff
@@ -348,13 +367,19 @@ void GUIFormSpecMenu::parseList(parserData* data, const std::string &element)
 
 		InventoryLocation loc;
 
-		if(location == "context" || location == "current_name")
+		if (location == "context" || location == "current_name")
 			loc = m_current_inventory_location;
 		else
 			loc.deSerialize(location);
 
-		v2s32 pos = getElementBasePos(true, &v_pos);
+		v2s32 pos;
 		v2s32 geom;
+
+		if (data->real_coordinates)
```

Comment:
I would prefer not to do this because there are so many irregularities and exceptions.  First off, there is the `getRealCoordinateGeometry`, which has no equivalent function for the older system, so the `if` would be required.  Also, there are some elements (like fields) which modify the position again after calling `getElementBasePos`.  The end result would likely be that the `if` would still be used nearly as much.

Reply:
Ok, this is fine then